plugins/modules/purefa_info.py

```diff
@@ -131,6 +131,7 @@
 POD_QUOTA_VERSION = "2.23"
 AUTODIR_API_VERSION = "2.24"
 SUBS_API_VERSION = "2.26"
+OBJECT_SAFE_MODE_API_VERSION = "2.26"
```

Comment:
You can use `SUBS_API_VERSION` as this already define 2.26

Reply:
I did see that there was an existing variable for the same API version, but the reason I had opted to have a separate line was for 2 reasons. First, it was more descriptive as to what the check was for. There was existing precedence to this as there are multiple variables for the `1.14` and `1.16` versions.

The other reason was that the `purefa_eradication` module most likely will also need to be updated to support the additional fields. I looked at it briefly and while it isn't broken like the `purefa_info` module is in this case, it does limit the options available. In the case the the `purefa_eradication` module is also updated, it would be convenient to have the same variable name across the modules for the same purpose where the name `SUBS_API_VERSION` wouldn't mean anything useful in the `purefa_eradication` module context.

All of that said, I am open to changing it to just use the `SUBS_API_VERSION`, I just figured it was worth the discussion.